lib/manager/cocoapods/artifacts.ts

```diff
@@ -106,7 +106,7 @@ export async function updateArtifacts({
 
   const podsDir = join(dirname(packageFileName), 'Pods');
   const podsManifestFileName = join(podsDir, 'Manifest.lock');
-  if (await platform.getFile(podsManifestFileName)) {
+  if (await readLocalFile(podsManifestFileName, 'utf8')) {
```

Comment:
```suggestion
  if (await readLocalFile(podsManifestFileName)) {
```

as we do not need result